lib/cli/options.ts

```diff
@@ -42,11 +41,11 @@ export class Option {
     if (value && typeof value === 'string') {
       return '' + value;
     }
-    return null;
+    return '';
```

Comment:
Some reason github was telling me that the line I was looking at was referring to `getNumber()`. I'm okay with this.